app.js

```diff
@@ -23,3 +24,31 @@ app.get('/', (req, res) => {
 });
 
 app.listen(3000, () => console.log('🏃‍ on port 3000'));
+
+app.get("/", (req, res) => {
+  res.render("")
+})
+
+app.get("/beers", (req,res)=> {
+   punkAPI.getBeers()
+  .then(beersFromApi => res.render("beers", {beersFromApi}))
+  .catch(error => console.log(error));
+})
+
+app.get("/randomBeer", (req, res)=> {
+  punkAPI.getRandom()
+  .then(responseFromAPI => res.render("randomBeer", {responseFromAPI}))
+  .catch(error => console.log(error))
+})
+
+
+app.get('/beer-detailed', (req, res) => {
+  res.render('beer-detailed')
```

Comment:
This one isn't working because if you go to your beer-detailed page, you only have a #each beerApi and the partial you created. If you want to display that, you would need to use the punkAPI in here.
So, for example:

```
punkAPI
  .getBeers()
  .then(beerApi => res.render('beer-detailed' , {beerApi}))
  .catch(error => console.log(error))
```